metabrainz/templates/index/team.html

```diff
@@ -0,0 +1,120 @@
+{% extends 'base.html' %}
+
+{% block title %}Team - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">Meet the MetaBrainz Foundation Team</h1>
+<div id="team" class="col-sm-12"> 
+ 
+  <div class="people">
+    <div class="pic long">
+      <img width="150px" height="150px" src="{{ url_for('static', filename='img/ruaok.jpg') }}">
+    </div>
+    <div>
+      <h4><strong>Executive Director</strong> - Robert Kaye | ruaok</h4>
+      <p>Robert is the founder and lead geek for <a href="https://musicbrainz.org/">MusicBrainz</a>, and Executive Director of the MetaBrainz Foundation, the non-profit organization that pays the bills for MusicBrainz. After studying Computer Engineering at Cal Poly, San Luis Obispo, he joined Xing Technology where he shaped the MP3 efforts and then spent time for the dot com boom on EMusic’s FreeAmp team. While at EMusic he started working on MusicBrainz, the open music encyclopedia, and fell in love with Open Source software.</p>
```

Comment:
Long strings should be wrapped. Pick a line length limit and stick to it. Let's say 100 characters. Also, when things inside a block don't fit into one line, they need to be separated from it like this:
```html
<p>
  Hello, I am
  a long line.
</p>
```